hypha/templates/includes/user_menu.html

```diff
@@ -1,67 +1,75 @@
 {% load i18n heroicons %}
 
 {% if request.user.is_authenticated %}
-    <div x-data="{ show: false }" class="relative flex">
+    <div x-data="{ show: false }" x-on:keydown.escape="show = false" class="relative flex">
         <button
             x-on:click="show = ! show"
-            class="font-bold flex items-center ps-1 pe-4 hover:bg-black hover:text-white transition-colors rounded-sm"
+            class="font-bold flex items-center transition-all rounded-sm px-1.5 bg-gray-100"
             type="button"
+            :class="show ? 'bg-gray-900 text-white' : 'hover:bg-light-blue/30'"
         >
-            {% heroicon_micro "user-circle" class="inline align-text-bottom w-8 h-8 me-1" aria_hidden=true %}
-            <span>{{ request.user }}</span>
+            {% heroicon_micro "user-circle" class="inline align-text-bottom w-8 h-8 md:me-1" aria_hidden=true %}
```

Comment:
```suggestion
            {% heroicon_micro "user-circle" class="inline group-hover:scale-110 transition-transform align-text-bottom w-8 h-8 md:me-1" aria_hidden=true %}
```
Just for more consistency with the notification bell, gives the cool icon scaling animation on hover.


https://github.com/HyphaApp/hypha/assets/145372368/dfb3966a-551b-4471-bd3e-93f405fabd54